docs/Releases.md

```diff
@@ -13,6 +13,326 @@ use a date based versioning system.  Now, a release version can look
+
+NF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop           out   /    tonf     /   drop
+               PNT / S|W / CHLD  drop_pps  /  drop_pps      rx_drop  /  tx_drop           next  /    buf      /   ret
+----------------------------------------------------------------------------------------------------------------------
+speed_tester    1  /  1  /  4      9661664 / 9661664        94494528 / 94494528               0 / 94494487    / 0
+                0  /  W  /  0            0 / 0                     0 / 0                      0 / 0           / 128
+```
+
+The shared core mode adds wakeup information stats:
+```
+PORTS
+-----
+Port 0: '90:e2:ba:b3:bc:6c'
+
+Port 0 - rx:         5  (        0 pps) tx:         0  (        0 pps)
+
+NF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop           out   /    tonf     /   drop
+               PNT / S|W / CHLD  drop_pps  /  drop_pps      rx_drop  /  tx_drop           next  /    buf      /   ret
+                                  wakeups  /  wakeup_rt
+----------------------------------------------------------------------------------------------------------------------
+simple_forward  2  /  2  /  4        27719 / 27719            764439 / 764439                 0 / 764439      / 0
+                0  /  S  /  0            0 / 0                     0 / 0                      0 / 0           / 0
+                                    730557 / 25344
+
+speed_tester    3  /  1  /  5        27719 / 27719            764440 / 764439                 0 / 764440      / 0
+                0  /  W  /  0            0 / 0                     0 / 0                      0 / 0           / 1
+                                    730560 / 25347
+
+
+
+Shared core stats
+-----------------
+Total wakeups = 1461122, Wakeup rate = 50696
+```
+
+
+
+### CI PR Review:
+CI is now available on the public branch. Only a specific list of whitelisted users can currently run CI due to security purposes. The new CI system is able to approve/reject pull requests.
+CI currently performs these checks:
+ - Check the branch (for our discussed change of develop->master as main branch)
+ - Run performance check (speed tester currently with 35mil benchmark)
+ - Run linter (only on the PR diff)
+
+### LPM Firewall NF:
+The firewall NF drops or forwards packets based on rules provided in the json file. This is achieved using DPDK's LPM (longest prefix matching) library. Default behavior is to drop a packet unless the packet matches a rule. The NF also has a debug mode to print decisions for every packet and an inverse match mode where default behavior is to forward a packet if it is not found in the table.
+
+Documentation for this NF can be found [here][firewall_nf_readme].
+
+
+### Payload Search NF:
+The Payload Scan NF provides the functionality to search for a string within a given UDP or TCP packet payload. Packet is forwarded to its destination NF on a match, dropped otherwise.
```

Comment:
A good first issue will be adding inverse match support to this

Reply:
Already there, I'll expand the description